src/main/java/com/conveyal/datatools/manager/controllers/api/DeploymentController.java

```diff
@@ -243,67 +245,69 @@ private static Object updateDeployment (Request req, Response res) {
      * Create a deployment bundle, and send it to the specified OTP target servers (or the specified s3 bucket).
      */
     private static String deploy (Request req, Response res) {
-        try {
```

Comment:
I can't recall why, but I'm pretty sure this try/catch with generic exception was added for a reason. I think it had to do with the fact that the name was being used for the deployment target ID (which will be addressed in #124). I guess we can remove this try/catch, but it makes me a little uneasy unless there is some safety net that handles exceptions thrown in spark controllers.